hoomd/hpmc/ComputeSDF.h

```diff
@@ -293,15 +315,17 @@ template<class Shape> void ComputeSDF<Shape>::countHistogram(uint64_t timestep)
     ArrayHandle<Scalar4> h_orientation(m_pdata->getOrientationArray(),
                                        access_location::host,
                                        access_mode::read);
-
+    ArrayHandle<Scalar> h_diameter(m_pdata->getDiameters(),
+                                   access_location::host,
+                                   access_mode::read);
+    ArrayHandle<Scalar> h_charge(m_pdata->getCharges(), access_location::host, access_mode::read);
```

Comment:
```suggestion

```
These are not used.

Reply:
They are used for the patch energy calculation.

Reply:
They are but not in this function that I can see.

Reply:
You're right. That's left over from my initial implementation where I didn't put the linear search in a separate method.